### PR TITLE
improve registration feedback

### LINKED ISSUE #25: Feedback for failed registration
There should be appropriate feedback if registration fails, especially for validation errors.
Currently the feedback is only "Something went wrong...".

### PR DESCRIPTION
**Pull request by dajenet**
closes #25

### CODE REVIEW

File: src/app/signup/signup.component.ts [hunk 1]
```diff
@@ -40,7 +40,13 @@ export class SignupComponent implements OnInit {
     this.accountService.register(this.user)
       .subscribe(
         () => this.router.navigateByUrl('/login?registered=true'),
-        () => this.wrongCredentials = true,
+        (error) => {
+          if (error.error.errors) {
```
Comment (vordemann) [inline src/app/signup/signup.component.ts:44]: Weird naming error.error.errors

**Review by vordemann — APPROVED** [review]
Does the job! Great. Just rethink the error.error.errors naming

File: src/app/signup/signup.component.ts (hunk 1 above)
Comment (dajenet) [reply src/app/signup/signup.component.ts:44]: The first error is the default naming for that function. The other are fixed from the json result. So would be registrationError.error.errors better? I am not sure.

Comment (chaoran-chen) [reply src/app/signup/signup.component.ts:44]: I think it's fine how it is. It's just funny :)

**Pull request merged**